### ISSUE TITLE
[bug?]: In Vue2 + elementUI 2.x, the component parameter prompt function unexpectedly inserted unnecessary additional content.

### ISSUE THREAD

**Issue by QuentinHsu** [Simon-He95/vscode-common-intellisense#44]
## System Info

```
Version: 1.85.1
Commit: 0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2
Date: 2023-12-13T09:48:16.874Z
Electron: 25.9.7
ElectronBuildId: 25551756
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Darwin arm64 23.2.0
```

## Plugin version
```
v0.1.0
```

## Reproduction steps

https://github.com/Simon-He95/vscode-common-intellisense/assets/21162238/de9ba021-800f-4771-9a74-aedbffab8afd

**Comment by Simon-He95** [comment]
Can you try uninstalling and reinstalling? It's normal here.

**Comment by QuentinHsu** [comment]
> Can you try uninstalling and reinstalling? It's normal here.

The problem still exists.

It is worth noting that the current discovery is with some props parameters, such as `close-on-click-modal` and `close-on-press-escape` of el-dialog, and `label-width` of el-form. The above situations can be reproduced 100%. I don't know if you have tried the above test cases.

**Comment by Simon-He95** [comment]
It seems that I have not encountered this problem. It doesn't matter. I just made some patches for other functions. You can try to upgrade to the latest version and try again.
![屏幕录制2024-01-09 18 43 29](https://github.com/Simon-He95/vscode-common-intellisense/assets/57086651/1d0c0451-458c-42ee-8b0b-8b7c638525a2)

**Comment by QuentinHsu** [comment]
I have tried the latest version `0.1.2` that you released, but the problem I encountered before still exists.

After looking at your GIF content, I found that there are still differences between your test cases and mine. So let me provide more details about my test case:

1. When all UI component tag contents are on one line, if an attribute accepts a non-string value, it will unexpectedly input additional content.
 - ✅:

https://github.com/Simon-He95/vscode-common-intellisense/assets/21162238/bdad42da-aeb6-4de0-a4a1-3bdb992f8d6b

  - ❌:

https://github.com/Simon-He95/vscode-common-intellisense/assets/21162238/2115698b-aa7a-48ab-acbc-f272080aa064

2. If there are too many attributes and they are subsequently formatted with line breaks in the code, additional content may be inadvertently inputted.

  - ❌:

https://github.com/Simon-He95/vscode-common-intellisense/assets/21162238/f1c5ad89-179b-4905-a09f-e37c7a82954e

Thank you very much for your hard work in trying to reproduce it again!
